.github/workflows/turing.yaml

```diff
@@ -24,6 +24,7 @@ on:
   pull_request:
     branches:
       - main
+      - grpc_upi # temporary enable CI when creating pr targeting grpc_upi branch
```

Comment:
This temporary change can be removed here since the target branch is `main`, right?

Reply:
Updated